packages/bits/src/lib/popover/popover.component.ts

```diff
@@ -446,6 +457,7 @@ export class PopoverComponent implements OnDestroy, OnInit, OnChanges {
             .flexibleConnectedTo(this.host.nativeElement)
             .withPush(false)
             .withViewportMargin(0)
+            .withGrowAfterOpen(true)
```

Comment:
Sets whether the overlay can grow after the initial open via flexible width/height.
--